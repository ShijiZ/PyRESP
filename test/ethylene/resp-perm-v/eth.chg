All values are reported in atomic units
%FLAG TITLE
 molecule 1: ethylene

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00   1.2441670E+00
   2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
   3   0.0000000E+00   1.7277940E+00   2.3159880E+00
   4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
   5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
   6   0.0000000E+00  -1.7277940E+00   2.3159880E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6          0    -1.6559495E-01
   2          6          1    -1.6559495E-01
   3          1          0     8.3307760E-02
   4          1          3     8.3307760E-02
   5          1          3     8.3307760E-02
   6          1          3     8.3307760E-02

%FLAG PERM DIP LOCAL: 3(I4,X5),I4,X2,E16.7
 dip.no   atm.no   ref.no   ivary      p(opt)
   1        1        2        0    -1.2059379E-01
   2        1        3        0     4.4398431E-02
   3        1        6        2     4.4398431E-02
   4        1        4        0    -9.5999537E-02
   5        1        5        4    -9.5999537E-02
   6        2        1        1    -1.2059379E-01
   7        2        4        2     4.4398431E-02
   8        2        5        2     4.4398431E-02
   9        2        3        4    -9.5999537E-02
  10        2        6        4    -9.5999537E-02
  11        3        1        0    -3.0333152E-01
  12        3        2        0     1.1629236E-01
  13        3        6        0     8.5102695E-02
  14        4        2       11    -3.0333152E-01
  15        4        1       12     1.1629236E-01
  16        4        5       13     8.5102695E-02
  17        5        2       11    -3.0333152E-01
  18        5        1       12     1.1629236E-01
  19        5        4       13     8.5102695E-02
  20        6        1       11    -3.0333152E-01
  21        6        2       12     1.1629236E-01
  22        6        3       13     8.5102695E-02

%FLAG PERM DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00   3.4013483E-01
   2   0.0000000E+00   0.0000000E+00  -3.4013483E-01
   3   0.0000000E+00   1.2188551E-01   5.5278477E-02
   4   0.0000000E+00  -1.2188551E-01  -5.5278477E-02
   5   0.0000000E+00   1.2188551E-01  -5.5278477E-02
   6   0.0000000E+00  -1.2188551E-01   5.5278477E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00  -9.4516997E-02
   2   0.0000000E+00   8.6736174E-19   9.4516997E-02
   3   0.0000000E+00  -3.1339371E-02  -5.9914915E-02
   4   0.0000000E+00   3.1339371E-02   5.9914915E-02
   5   0.0000000E+00  -3.1339371E-02   5.9914915E-02
   6   0.0000000E+00   3.1339371E-02  -5.9914915E-02

