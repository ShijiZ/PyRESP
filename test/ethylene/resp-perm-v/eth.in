test for ethylene
 &cntrl
 nmol = 1,
 iqopt = 1,
 ihfree = 1,
 qwt = 0.0005,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 1,
 pwt = 0.0005,
 virtual = 1
 &end 
    1.0
ethylene
   -99   6   22  
    6    0   0  0  2  0  4
    6    1   1  2  2  4  4
    1    0   0  0  0
    1    3   11 12 13
    1    3   11 12 13
    1    3   11 12 13




