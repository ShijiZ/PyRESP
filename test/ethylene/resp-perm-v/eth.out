
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for ethylene
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00050000
 virtual     = 1

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: ethylene
 Total charge (ich):-99
 Number of centers:  6
 Number of permanent dipoles: 22
    1    6    0    0    0    2    0    4
    2    6    1    1    2    2    4    4
    3    1    0    0    0    0
    4    1    3   11   12   13
    5    1    3   11   12   13
    6    1    3   11   12   13

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    6    0    0    0    2    0    4
    6    1    1    2    2    4    4
    1    0    0    0    0
    1    3   11   12   13
    1    3   11   12   13
    1    3   11   12   13

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =    6
 Total number of permanent dipoles      =   22
 Weight factor on initial charge restraints=  0.000500
 Weight factor on initial dipole restraints=  0.000500
 There are  0 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =     6
 total number of esp points =   452

 center       X               Y               Z
    1   0.0000000E+00   0.0000000E+00   1.2441670E+00
    2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
    3   0.0000000E+00   1.7277940E+00   2.3159880E+00
    4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
    5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
    6   0.0000000E+00  -1.7277940E+00   2.3159880E+00


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 and p0 values will be set to 0

 Number of unique UNfrozen charge centers =     2
 Number of unique UNfrozen permanent dipoles =     6

 Non-linear optimization requested.
  qchnge =    8.5902233167E-02
  qchnge =    2.0139790664E-02
  qchnge =    8.3807195508E-03
  qchnge =    4.3030834919E-03
  qchnge =    2.3214295491E-03
  qchnge =    1.2733384791E-03
  qchnge =    7.0347839656E-04
  qchnge =    3.9009152750E-04
  qchnge =    2.1679344069E-04
  qchnge =    1.2066656306E-04
  qchnge =    6.7241333258E-05
  qchnge =    3.7506981437E-05
  qchnge =    2.0939530895E-05
  qchnge =    1.1699604301E-05
  qchnge =    6.5419077119E-06
  qchnge =    3.6605910131E-06
  qchnge =    2.0497399678E-06
  qchnge =    1.1485123191E-06
  qchnge =    6.4394885494E-07

 Convergence in   18 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: ethylene

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.165595      0       0.002585
    2         6        0.000000     -0.165595      1       0.002585
    3         1        0.000000      0.083308      0       0.000000
    4         1        0.000000      0.083308      3       0.000000
    5         1        0.000000      0.083308      3       0.000000
    6         1        0.000000      0.083308      3       0.000000

 Sum over the calculated charges:      0.002

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         6        0.000000     -0.120594      0       0.003192
    2         6        0.000000      0.044398      0       0.004570
    3         6        0.000000      0.044398      2       0.004570
    4         6        0.000000     -0.096000      0       0.003607
    5         6        0.000000     -0.096000      4       0.003607
    6         6        0.000000     -0.120594      1       0.003192
    7         6        0.000000      0.044398      2       0.004570
    8         6        0.000000      0.044398      2       0.004570
    9         6        0.000000     -0.096000      4       0.003607
   10         6        0.000000     -0.096000      4       0.003607
   11         1        0.000000     -0.303332      0       0.000000
   12         1        0.000000      0.116292      0       0.000000
   13         1        0.000000      0.085103      0       0.000000
   14         1        0.000000     -0.303332     11       0.000000
   15         1        0.000000      0.116292     12       0.000000
   16         1        0.000000      0.085103     13       0.000000
   17         1        0.000000     -0.303332     11       0.000000
   18         1        0.000000      0.116292     12       0.000000
   19         1        0.000000      0.085103     13       0.000000
   20         1        0.000000     -0.303332     11       0.000000
   21         1        0.000000      0.116292     12       0.000000
   22         1        0.000000      0.085103     13       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          2.0365103E-02
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   2.9116308E-04
 The root-mean-squared error (RMSE = sqrt(RSS/N))               8.0259959E-04
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.1957072E-01

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        0.00000    0.00000    0.00000   -0.00000

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   -2.08819
        Y    0.00000    0.93324
        Z    0.00000    0.00000    1.15495

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X    1.15495
        Y    0.00000    0.93324
        Z    0.00000    0.00000   -2.08819